static/js/components/PodcastPlayButton.js

```diff
@@ -56,23 +56,25 @@ export default function PodcastPlayButton(props: Props) {
   const className = episodeCurrentlyPlaying ? "grey-surround" : "black-surround"
 
   return (
-    <div
-      className={`podcast-play-button ${className}`}
-      onClick={
-        episodeAudioInitialized ? togglePlayState : initializeAudioPlayer
-      }
-    >
-      {episodeCurrentlyPlaying ? (
-        <>
-          Pause
-          <i className="material-icons pause">pause</i>
-        </>
-      ) : (
-        <>
-          Play
-          <i className="material-icons play_arrow">play_arrow</i>
-        </>
-      )}
+    <div className={`podcast-play-button ${className}`}>
+      <a
+        href="#"
```

Comment:
I think you can leave this off since we are already calling `preventDefault` in the onClick handler